test/index-test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const chai = require('chai');
+const main = require('../index.js');
+const assert = chai.assert;
+
+describe('testing main module', function(){
+  describe('testing main with args', function(){
+    before(() => {
```

Comment:
Great job testing for the bonus! This isn't related to any assignment requirements, but just an FYI that your before/after block don't seem to be necessary in this case. 

Your main() function in index.js wll always pass in "gnoevil" if an arg isn't passed into the command line.